tests/functional/test_sync_wrap.py

```diff
@@ -130,6 +135,32 @@ def _is_rate_limited_polling(response):
         assert False, "Timeout Error: Rate limit with sync wrap request wasn't tripped within set timeout"
 
 
+@pytest.mark.sync_wrap
+@when("the rate limit is tripped with sync-wrap polling the create Patient endpoint")
+def trip_rate_limit_sync_polling_create_patient(context: dict):
+    context["pds"].headers = {
+         "X-Sync-Wait": "29"
+    }
+
+    def _create_patient():
+        response = context["pds"].post(
+            data={"nhsNumberAllocation": "Done"},
+            headers=context["pds"].headers,
+            url=f'{context["pds"].base_url}/Patient/'
+        )
+        return response
+
+    def _is_rate_limited_polling(response):
+        return response.status_code == 429
```

Comment:
Might be worth check that the 429 rejection is for the SpikeArrest.PatientCreate policy rather than another spike arrest policy. Maybe checking the response body for text?

Reply:
@davidhamill1-nhs We have the step "then returns a rate limit error message" for this check. At the moment, I have `assert code == "TOO_MANY_REQUESTS"`

Reply:
The code `TOO_MANY_REQUESTS` is also returned by the app- and proxy-based spike arrest policies. Given previous incarnations of this test passed because they triggered this rate limiting policy, it might be worth checking something distinct to the Patient Create response.

Reply:
I agree, we need the testing to definitively prove it's our spike arrest policy being triggered